snapshots/education/2018-04-18/literacy_rates.feather.dvc

```diff
@@ -1,82 +1,27 @@
-
-
-      - Data for 1820 and 1870 (except for the US): Broadberry  and O'Rourke (2010)
-      – The Cambridge Economic History of Modern
-      Europe: Volume 1, 1700-1870
-
-
-      - Data for the US: National Center for Education Statistics, online [here](nces.ed.gov/naal/lit_history.asp).
-
-
-      - Global estimates for 1820-2000: van Zanden, J.L., et al. (eds.) (2014), How
-      Was Life?: Global Well-being since 1820,
-      OECD Publishing. Online [here](http://www.oecd.org/statistics/how-was-life-9789264214262-en.htm).
-
-
-      - Historical estimates for Latin America: OxLAD – Oxford Latin American Economic
-      History Data Base, online [here](http://moxlad-staging.herokuapp.com/home/en).
-
-
-      - Most recent estimates for high-income countries, as well as any available
-      estimates for 2016: [CIA World Factbook](https://www.cia.gov/library/publications/the-world-factbook/).
-
-
-
-
-      Further notes:
-
-
-      - All sources rely on the same conceptual definition, but in many cases sources
-      do not agree with one another. Because
-      of this, year to year changes should be interpreted with caution. You can read
-      more about literacy measurement here:
-      https://ourworldindata.org/how-is-literacy-measured
-
-
-      - The World Bank's WDI estimates correspond to UNESCO Institute for Statistics.
-      OxLAD estimates come from several underlying
-      sources (see original documentation for more details).
-
-
-      - For Paraguay in 1982, the OxLAD data source was favoured over the World Bank
-      (WDI) as the latter estimates literacy
-      rates at 78.46%, a much lower estimate compared to neighbouring years for which
-      there was data available.
-
-
-      - Sources for each country-year observation can be found in [this table.](http://ourworldindata.org/wp-content/uploads/2018/06/cross-country-literacy-sources-final.csv)
-    url: '- World Bank: https://data.worldbank.org/indicator/SE.ADT.LITR.ZS </br>
-      - CIA Factbook: https://www.cia.gov/library/publications/the-world-factbook/
-      </br> - Links to other sources below'
+  origin:
+    # Data product / Snapshot
+    title: Cross-country literacy rates - World Bank, CIA World Factbook, and other Sources
+    description: >
+      This dataset combines long-run cross-country literacy rates from several sources
+      including the World Bank's WDI, CIA Factbook, and other historical data series.
+      Key sources include Buringh & Van Zanden (2009) for data before 1800, Broadberry
+      and O'Rourke (2010) for 1820 and 1870, and van Zanden et al. (2014) for global
+      estimates from 1820-2000. It also includes recent estimates for high-income
+      countries and available 2016 data from the CIA World Factbook. Note that the
+      sources may have discrepancies and different underlying methodologies, and thus
+      year-to-year changes should be interpreted cautiously.
+    # Citation
+    producer: Our World in Data
```

Comment:
As explicitly mentioned in [the guidelines](https://docs.owid.io/projects/etl/architecture/metadata/reference/origin/#originproducer), and discussed multiple times, the producer is (almost) never OWID. The only case where this is true is for the `regions` dataset. If you have very strong reasons to choose OWID as a producer, it would be good to discuss it and change the guidelines.
In this case, I suppose the producer could be either "World Bank" or "Various sources".